configs/promtail/promtail-config.yml

```diff
@@ -0,0 +1,23 @@
+clients:
+  - url: http://loki:3100/loki/api/v1/push
+
+scrape_configs:
+- job_name: syslog
+  syslog:
+    listen_address: 0.0.0.0:1514
+    listen_protocol: tcp
+    idle_timeout: 300s
+    label_structured_data: yes
+    labels:
+      job: "syslog"
+  relabel_configs:
+    - source_labels: ['__syslog_message_hostname']
+      target_label: 'source'
+    - source_labels: ['__syslog_message_severity']
+      target_label: 'level'
+    - source_labels: ['__syslog_message_app_name']
+      target_label: 'application'
+    - source_labels: ['__syslog_message_facility']
+      target_label: 'facility'
+    - source_labels: ['__syslog_message_proc_id']
+      target_label: 'procid'
```

Comment:
@KTodts Is this required, do you remember from top of your head?

Reply:
I am merging this, but would be cool for you to clarify that for our future post

Reply:
@hellt, this is creating the labels used in loki based on the incoming syslog message. The labels are used for filtering in the dashboard.

You can search in loki db the same way you do it in prometheus with labels, e.g {application="sr_cli", source="leaf1"}

Reply:
which piece adds the `__syslog_message_*` label? Is it syslogng or default promtail labelling scheme?

Reply:
Promtail creates those labels based on incoming syslog messages according to RFC5424. Its not syslogng setting it. All available promtail labels can be found [here](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#available-labels)